### DISCUSSION TITLE
Remote changed IP address

### DISCUSSION THREAD

**Discussion by ryanmsteed** [JackJPowell/hass-unfoldedcircle#74] [Q&A]
Hey there Jack and community. Love the integration! Is there a way to manually change the host IP it's looking for? The remote got a new IP for some reason and broke the integration. If I try to add a new device I get the unknown error, perhaps because I haven't deleted the old device? Hoping to avoid that.. Any info would be appreciated!

**Comment by JackJPowell** [comment]
As it stands, there isn't a way to update the Host/IP of the remote in Home Assistant. However, if you didn't update names/entity ids, deleting and re-adding should be very painless. 

Curious, do you recall if you added it manually (supplying an IP) or if you added it via zeroconf discovery? 

I'm away on vacation so I won't be able to update things for a week or two, but I'll make this change in a future release

**Comment by ryanmsteed** [comment]
Appreciate the response while on vacation! You are correct - deleting and re-adding worked great. And when I re-added, it looked very familiar so I think I did indeed use the zeroconf discovery last time. Happy to do some testing if you need when you're back from vacation. Thanks again for the help!

**Comment by JackJPowell** [comment]
An option has been added to the reconfigure flow that allows the user to update the host / ip address of the remote in beta 12 :)

If you are running the remote firmware version 2.2.2 and haven't updated the home assistant integration to beta 14 yet, I have something for you to test for me. No pressure if you don't want to update to either yet though.

**Reply by ryanmsteed** [reply]
Thanks Jack! Will try to check it out this week

**Comment by JackJPowell** [comment]
Version 14 has been released as well as a couple of follow on point releases that include the ability to reconfigure your remote's address.
